packages/fluentui/docs/src/utils/calculateBreakpoints.ts

```diff
@@ -0,0 +1,28 @@
+export type Breakpoints = Record<number, number> & { default?: number };
```

Comment:
can you move it to the same folder as the prototype? I don't think `docs/*` is exported and this is very specific to the prototype

Reply:
Done